dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -498,6 +498,12 @@
       # This setting can also be overridden using the environment variable ZEEBE_BROKER_EXPERIMENTAL_MAXAPPENDBATCHSIZE
       # maxAppendBatchSize = 32KB;
 
+      # When this flag is enabled, the leader election algorithm attempts to elect the leaders based on a pre-defined priority.
```

Comment:
Maybe we add a table here how it would look like? Or is it too much?

Reply:
We cannot guarantee that the leadership will be distributed exactly as expected. So I think it would be better to not give any expectations by documenting it.

Reply:
I more thought about the assigned priorities, which will increase the change to become leader. But ok :)